### ISSUE TITLE
Asus, Kinect coordinate system definitions might not match in cob_openni2_tracker

### ISSUE THREAD

**Issue by ipa-rmb** [ipa320/cob_people_perception#59]
It is possible that the Asus Xtion and the Kinect sensor and maybe even the openni_launch and openni2_launch drivers output data with different definitions of coordinate systems (or at least in their internal representations). If that is true, the cob_openni2_tracker software needs to identify the utilized device or driver and convert coordinates accordingly. The current setting is good for Asus Xtion with openni2_launch. There is no Kinect available for testing at this moment.

**Comment by rpapallas** [comment]
We have a Kinect 2 (Xbox One) available. We are interested making this working with the project, so we are available to provide any testing required to make this happen.

**Comment by ipa-rmb** [comment]
If somebody is using the cob_openni2_tracker software together with a Kinect and observes that the point cloud with detected people does not match the RGB-D point cloud from the sensor, please report me that issue together with a screenshot from Rviz showing both point clouds and tf.

**Comment by ipa-rmb** [comment]
Yes you might try. I am not sure, though, whether openni2_tracker is meant to work with that hardware, because it is literally a completely different sensor. To my knowledge, the tracker goes deep into the hardware driver of the Asus or Kinect 1 device. So you might not be successful with a Kinect 2 and this software. But feedback is welcome.

**Comment by rpapallas** [comment]
@ipa-rmb Of course. I may be able to give some information about it later this month.

**Comment by fmessmer** [comment]
https://github.com/ipa320/cob_robots/issues/669 might be related

**Comment by fmessmer** [comment]
asus coordinate systems have been fixed in https://github.com/ipa320/cob_robots/pull/720 and https://github.com/ipa320/cob_common/pull/233

@ipa-rmb @cagbal could you retest and verify?
